pool/test/custom_field_test.ml

```diff
@@ -242,6 +248,40 @@ let update () =
       events)
 ;;
 
+let update_type_of_published_field () =
+  let open CCResult in
+  let custom_field =
+    Data.custom_text_field
+      ~published_at:(Custom_field.PublishedAt.create_now ())
+      ()
+  in
+  let events =
+    let data =
+      Message.
+        [ (Field.(FieldType |> show), Custom_field.FieldType.(Number |> show))
+        ; Field.(AdminHint |> show), Data.admin_hint
+        ]
+      |> CCList.map (fun (f, l) -> f, l |> CCList.pure)
```

Comment:
What's the purpose of this map? Add the `[]` directly to the second element above.